src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Headers.cs

```diff
@@ -21,6 +22,8 @@ public abstract class HttpClientHandlerTest_Headers : HttpClientHandlerTestBase
     {
         public HttpClientHandlerTest_Headers(ITestOutputHelper output) : base(output) { }
 
+        internal sealed class DerivedHttpHeaders : HttpHeaders { }
```

Comment:
nit: I think this can be 'private' instead of 'internal' since it is a nested class and only used in this test class.